wiki/Bazel-Setup-Instructions-for-Linux.md

```diff
@@ -0,0 +1,67 @@
+## Instructions
+
+**The steps to install Bazel on Linux are:**
+1. Install Bazel 
+2. Install the OpenJDK 8
```

Comment:
Drop "the", here and in the header below.

Reply:
Done